README.md

```diff
@@ -77,60 +77,145 @@ Notes:
+Note that it is the file hashes of .sig and .sha256 that *verify* compares, not their raw contents.
+
+</p>
+</details>
+<br>
+
+Now to determine ***who*** the Public key ID belongs to: Goto [Keybase.io/seedsigner](www.keybase.io/seedsigner)  
+<BR>
+![SS - Keybase Website PubKey visual matching1_Cropped-80pct](https://user-images.githubusercontent.com/91296549/215326193-97c84e35-5570-4e52-bf3f-e86d367c8908.jpg)
+
+
+
+**You must now *manually* compare: The 16 character fingerprint ID (as circled in red above) to, those *rightmost* 16 characters from your *verify* command.** 
+
+**If they match exactly, then you have successfully confirmed that your .sig file is authentically from the Seedsigner Project!**
+<BR>
+
+<details><summary>Learn more about how keybase.io helps you check that someone (online) is who they say they are:</summary>
+<p>
+Keybase.io allows you to independently verify that the public key saved on Keybase.io, is both authentic and that it belongs to the organization it claims to represent.  
+ Keybase has already checked the three pubkey file locations  cryptographically when they were saved there. You can further verify the key publications if you would like:  
+ 
+ - *via Keybase*: By clicking on any of the three blue badges to see that the "proof" was published at that location. (The blue badge marked as tweet, is in the most human-readable form and it is also a bi-directional link on Twitter)    
+or, 
+ - *without keybase (out-of-band)*: By using these 3 links directly: [Twitter](https://twitter.com/seedsigner/status/1530555252373704707), [Github](https://gist.github.com/SeedSigner/5936fa1219b07e28a3672385b605b5d2) and [Seedsigner.com](https://seedsigner.com/keybase.txt). This method can be used if you would like to make an even deeper, independent inspection without relying on Keybase at all, or if the Keybase.io site is no longer valid or it is removed entirely. 
+
+Once you have used one of these methods, you will know if the Public Key stored on Keybase, is genuinely from the SeedSinger Project or not.
+</p>
+</details>
+<br>
+
+If the two ID's do *not* match, then you must stop here immediately. Do not continue. Contact us for assistance in the Telegram group address above.
+
+<br>
+
+### Step 2. Verifying that the *software images/binaries* are genuine
+
+Now that you have confirmed that you do have the real Seedsigner Project's Public Key (ie the 16 characters match) - you can return to your terminal window. Running the the *shasum* command, is the final verification step and will confirm (via file hashing) that the software code/image files (ie the binary files inside the zip file), were also not altered since publication, or even during your download process.
+
+ **On Linux or OSX:** Run this command
 ```
-Good signature from "seedsigner <btc.hardware.solutions@gmail.com>" [unknown]
+shasum -a 256 --ignore-missing -check seedsigner_0_*_*.img.zip.sha256
```

Comment:
I missed this on my first pass, `-check` need to be be `--check`

Reply:
Thx Nick, great catch! The "--check" is fixed now. I will work on the _Writing to MicroSD_ section today. 
I will keep it short, and just emphasize doing the "verify image" step to avoid those occasional errors users pick up when the device wont boot/nothing displays onscreen.